.circleci/config.yml

```diff
@@ -26,7 +40,17 @@ jobs:
           name: Java Linter
           command: |
             bazel build //:google_java_format_verification
-
+  license-header:
+    docker:
+      - image: circleci/golang:1.13
+    working_directory: /home/circleci/workspace/gapic-generator-java
+    steps:
+      - checkout
+      - run:
+          name: License Header
+          command: |
+            go get -u github.com/google/addlicense
+            addlicense -c "Google LLC" -l apache .
```

Comment:
This should be a check on Java-only files as well, so we don't accidentally modify files that shouldn't contain the header.

Reply:
done.